src/controllers/controllerengine.cpp

```diff
@@ -886,7 +881,7 @@ QScriptValue ControllerEngine::connectControl(
    Output:  true if successful
    -------- ------------------------------------------------------ */
 void ControllerEngine::disconnectControl(const ControllerEngineConnection conn) {
-    ControlObjectSlave* cos = getControlObjectSlave(conn.key.group, conn.key.item);
+    ControlObjectScript* cos = getControlObjectScript(conn.key.group, conn.key.item);
```

Comment:
I would prefer the name "coScript" over "cos".

(The "p" prefix for a pointer does not improve the readability in this narrow context, so I would omit it as before)